pkg/agent/openflow/pipeline.go

```diff
@@ -1807,16 +1886,20 @@ func (f *featurePodConnectivity) gatewayIPSpoofGuardFlows() []binding.Flow {
 	var flows []binding.Flow
 	for _, ipProtocol := range f.ipProtocols {
 		targetTable := SpoofGuardTable.ofTable.GetNext()
+		ctZoneTypeRegMark := IPCtZoneTypeRegMark
 		if ipProtocol == binding.ProtocolIPv6 {
 			targetTable = IPv6Table.ofTable.GetID()
+			ctZoneTypeRegMark = IPv6CtZoneTypeRegMark
 		}
-		flows = append(flows,
-			SpoofGuardTable.ofTable.BuildFlow(priorityNormal).
-				Cookie(cookieID).
-				MatchProtocol(ipProtocol).
-				MatchInPort(config.HostGatewayOFPort).
-				Action().ResubmitToTables(targetTable).
-				Done(),
+		fb := SpoofGuardTable.ofTable.BuildFlow(priorityNormal).
+			Cookie(cookieID).
+			MatchProtocol(ipProtocol).
+			MatchInPort(config.HostGatewayOFPort)
+		if f.connectUplinkToBridge {
+			fb = fb.Action().LoadRegMark(ctZoneTypeRegMark)
```

Comment:
Could you add comments to explain why this is needed?

Reply:
Added comment here.
In general, all IP/IPv6 traffic's `ctZoneTypeRegMark` must be set for conntrack use.